src/vs/workbench/services/textfile/common/textFileEditorModelManager.ts

```diff
@@ -283,7 +283,7 @@ export class TextFileEditorModelManager extends Disposable implements ITextFileE
 		this.mapResourceToModel.clear();
 		this.mapResourceToPendingModelLoaders.clear();
 
-		// dispose dispose listeners
```

Comment:
This is actually disposing dispose listeners.

Reply:
Thanks for clarifying, I wasn't to sure about this one